cmd/swarm/flags.go

```diff
@@ -64,6 +64,11 @@ var (
 		Usage:  "SWAP chequebook contract address",
 		EnvVar: SwarmEnvChequebookAddr,
 	}
+	SwarmSwapChequebookFactoryFlag = cli.StringFlag{
+		Name:   "swap-chequebook-factory",
+		Usage:  "address for the chequebook factory",
```

Comment:
Minor: I think address of the chequebook factory is better

Reply:
Agreed